tests/functional/SqlSyncTest.php

```diff
@@ -25,58 +25,68 @@ public function testSimulatedSqlSync()
         }
 
         $fixtureSites = [
-            'remote' => [
-                'host' => 'server.isp.simulated',
-                'user' => 'www-admin',
-                'ssh' => [
-                    'options' => '-o PasswordAuthentication=whatever'
+                'remote' => [
+                        'host' => 'server.isp.simulated',
+                        'user' => 'www-admin',
+                        'ssh' => [
+                                'options' => '-o PasswordAuthentication=whatever',
+                        ],
+                        'paths' => [
+                                'drush-script' => '/path/to/drush',
+                        ],
                 ],
-                'paths' => [
-                    'drush-script' => '/path/to/drush',
+                'local' => [
                 ],
-            ],
-            'local' => [
-            ],
         ];
         $this->setUpSettings($fixtureSites, 'synctest');
         $options = [
-            'uri' => 'OMIT',
-            'simulate' => null,
-            'alias-path' => __DIR__ . '/resources/alias-fixtures',
+                'uri' => 'OMIT',
+                'simulate' => null,
+                'alias-path' => __DIR__ . '/resources/alias-fixtures',
         ];
 
         $expectedAliasPath = '--alias-path=__DIR__/resources/alias-fixtures';
 
         // Test simulated simple rsync remote-to-local
-        $this->drush('sql:sync', ['@synctest.remote', '@synctest.local'], $options, '@synctest.local');
+        $this->drush('sql:sync', [
+                '@synctest.remote',
```

Comment:
4 space indent is sufficient, unless our PHPCS is complaining about it.

Reply:
Yes, it's complaining. I left that as it is.

Reply:
I locally changed these from 8 indent to 4 indent and `composer lint` still passed. Indentation needs fixing before merge please.